q2_sample_classifier/utilities.py

```diff
@@ -694,7 +694,7 @@ def _mean_feature_importance(importances):
     containing importance scores of the same features from multiple models
     (e.g., CV importance scores).
     '''
-    imp = pd.concat(importances, axis=1, sort=True)
+    imp = pd.concat(importances, axis=1)
```

Comment:
Do we need to revert this change?

Reply:
perhaps not, but I told @Oddant1 to just create a revert commit

Reply:
my thinking was we could double-revert later